### PR TITLE
chore(build): remove travis.yml file

### PR DESCRIPTION
**Pull request by shubham14bajpai**
Signed-off-by: shubham <shubham.bajpai@mayadata.io>

The tests and the builds have been moved to github actions. This PR removes the travis.yml file.

### CODE REVIEW

**Review by akhilerm — COMMENTED** [review]
@shubham14bajpai Can you also remove the occurrence of travis variable like `TRAVIS_BRANCH` , `TRAVIS_TAG` etc from the push script and makefile

**Comment by akhilerm** [comment]
Also, the build badges in README can be replaced with [Build Status](https://github.com/openebs/cstor-operators/actions/workflows/build.yml)

File: build/push [hunk 1]
```diff
@@ -107,12 +107,12 @@ then
   # This unique/build image will be pushed to corresponding ci repo.
   TagAndPushImage "${DIMAGE}-ci" "${BUILD_TAG}"
 
-  if [ ! -z "${TRAVIS_TAG}" ] ;
+  if [ ! -z "${RELEASE_TAG}" ] ;
   then
     # Push with different tags if tagged as a release
     # When github is tagged with a release, then Travis will
```
Comment (akhilerm) [inline build/push:113]: Can you change this also, 
```suggestion
    # When github is tagged with a release, then github actions reelase workflow will
```

**Review by akhilerm — COMMENTED** [review]
Basically we want to remove all occurrence of travis from the repository

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # cstor-operators
 [![Go](https://github.com/openebs/cstor-operators/workflows/Go/badge.svg)](https://github.com/openebs/cstor-operators/actions)
```
Comment (akhilerm) [inline README.md:2]: What is this badge for?

Comment (akhilerm) [reply README.md:2]: cc: @shubham14bajpai

Comment (shubham14bajpai) [reply README.md:2]: Go report

Comment (shubham14bajpai) [reply README.md:2]: Okay not for go report but earlier we had an action named go. I will remove this.

**Review by akhilerm — APPROVED** [review]
LGTM

**Review by prateekpandey14 — APPROVED** [review]

**Pull request merged**